components/FlexibleHeader/src/MDCFlexibleHeaderView.m

```diff
@@ -16,16 +16,15 @@
 
 #import "MDCFlexibleHeaderView.h"
 
+#import "MaterialApplication.h"
```

Comment:
This looks out of place - remove?

Reply:
Alternatively, if it's required, add `    ss.dependency "MaterialComponents/private/Application"` to the podspec.

Reply:
We need it in order to get the status bar height ([UIApplication mdc_safeSharedApplication])

Reply:
Looks like it's already there? https://github.com/material-components/material-components-ios/blob/develop/MaterialComponents.podspec#L255